bin/cml/pr.js

```diff
@@ -1,79 +1,13 @@
-const kebabcaseKeys = require('kebabcase-keys');
-
-const {
-  GIT_REMOTE,
-  GIT_USER_NAME,
-  GIT_USER_EMAIL,
-  repoOptions
-} = require('../../src/cml');
+const { options, handler } = require('./pr/create');
 
 exports.command = 'pr <glob path...>';
-exports.description = 'Create a pull request with the specified files';
-
-exports.handler = async (opts) => {
-  const { cml, globpath: globs } = opts;
-  const link = await cml.prCreate({ ...opts, globs });
-  console.log(link);
-};
-
+exports.description = 'Manage pull requests';
+exports.handler = handler;
 exports.builder = (yargs) =>
-  yargs.env('CML_PR').options(
-    kebabcaseKeys({
-      ...repoOptions,
-      md: {
-        type: 'boolean',
-        description: 'Output in markdown format [](url).'
-      },
-      skipCI: {
-        type: 'boolean',
-        description: 'Force skip CI for the created commit (if any)'
-      },
-      merge: {
-        type: 'boolean',
-        alias: 'auto-merge',
-        conflicts: ['rebase', 'squash'],
-        description: 'Try to merge the pull request upon creation.'
-      },
-      rebase: {
-        type: 'boolean',
-        conflicts: ['merge', 'squash'],
-        description: 'Try to rebase-merge the pull request upon creation.'
-      },
-      squash: {
-        type: 'boolean',
-        conflicts: ['merge', 'rebase'],
-        description: 'Try to squash-merge the pull request upon creation.'
-      },
-      branch: {
-        type: 'string',
-        description: 'Branch name for the pull request.'
-      },
-      title: {
-        type: 'string',
-        description: 'Pull request title.'
-      },
-      body: {
-        type: 'string',
-        description: 'Pull request description.'
-      },
-      message: {
-        type: 'string',
-        description: 'Commit message.'
-      },
-      remote: {
-        type: 'string',
-        default: GIT_REMOTE,
-        description: 'Sets git remote.'
-      },
-      userEmail: {
-        type: 'string',
-        default: GIT_USER_EMAIL,
-        description: 'Sets git user email.'
-      },
-      userName: {
-        type: 'string',
-        default: GIT_USER_NAME,
-        description: 'Sets git user name.'
-      }
-    })
-  );
+  yargs
+    .commandDir('./pr', { exclude: /\.test\.js$/ })
+    .recommendCommands()
+    .env('CML_PR')
```

Comment:
Why the others do not need it?

Reply:
Because `cml pr` already exists as a legacy command, an we can't break compatibility.

Reply:
But the others had the env to right?
Look [this](https://github.com/iterative/cml/pull/1073#pullrequestreview-1079107987)

Reply:
Others don't need that `.env()` because it was moved to the verbs (sub-sub-commands), but those that share the same name as the legacy commands still need a separate `.env()` methinks.